incubator/opa/templates/ca-updater-cronjob.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "opa.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "opa.name" . }}
+    helm.sh/chart: {{ include "opa.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  schedule: {{ .Values.caUpdater.schedule | quote }}
+  jobTemplate:
+    metadata:
+      name: {{ include "opa.fullname" . }}
+      labels:
+        app.kubernetes.io/name: {{ include "opa.name" . }}
+        helm.sh/chart: {{ include "opa.chart" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          serviceAccountName: {{ include "opa.name" . }}-ca-updater
+          containers:
+          - name: fix-validating-cabundle
+            image: quay.io/sudermanjr/utilities:latest
+            command: ["bash"]
+            args:
+              - -c
+              - |
+                cat > patch <<EOF
+                webhooks:
+                  - name: validating-webhook.openpolicyagent.org
+                    clientConfig:
+                      caBundle: $(kubectl -n opa get secret opa-ca-certificate -ojsonpath='{.data.ca\.crt}' | tr -d '\n')
```

Comment:
Should this secret have `{{ include "opa.fullname" . }}` as the prefix?

Reply:
Nice catch.